tests/Feature/ContactTest.php

```diff
@@ -28,6 +28,19 @@ private function fetchUser()
         return [$user, $contact];
     }
 
+    public function test_user_can_search_contacts()
+    {
+        $user = $this->signIn();
+        $contact = new Contact;
+        $firtContact = $contact->all()->first();
+        $response = $this->call('POST', 'people/search', [
+            '_token' => csrf_token(),
```

Comment:
`_token` is not needed to run the test.

Reply:
Okey, Thanks Asbiin.
I am new to Test Case so I may not to good with Testcases.